cucumber/FreeCourseEnrollment.feature

```diff
@@ -0,0 +1,25 @@
+@i9n
+	Feature: assinatura de um curso gratuito
+	
+	As a usuário comum
+	I want to assinar um curso gratuito
+	So that eu posso consumir sem custo o conteúdo do curso
+
+	Scenario: assinatura de um curso
+	Given estou me cadastrando em um curso
+	And o curso selecionado foi cadastrado como “Gratuito”
+	When eu confirmo a inscrição no curso
+	Then a inscrição é automaticamente aprovada
+
+	Scenario: cancelamento de um curso anteriormente “Gratuito”
+	Given estou inscrito em um curso que está configurado como “Gratuito”
+	And a configuração de pagamento é mudada para “Assinatura Mensal”
```

Comment:
Se no momento da inscrição o curso era gratuito, não importa ele ter mudado para assinatura mensal, o cancelamento será sem custo, certo? Se for assim, esse step poderia ser eliminado e a coisa toda ficaria assim:
Scenario: cancelamento de curso originalmente “Gratuito”
Given eu estou inscrito em um curso originalmente configurado como “Gratuito”
When eu escolho cancelar o curso
Then minha inscrição no curso é cancelada